WalletWasabi/JsonConverters/UnblindedSignatureJsonConverter.cs

```diff
@@ -20,19 +20,30 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 		{
 			JArray arr = JArray.Load(reader);
 
-			string c = arr[0].Value<string>();
-			string s = arr[1].Value<string>();
+			var cs = BigInteger.Parse(arr[0].Value<string>());
+			var ss = BigInteger.Parse(arr[1].Value<string>());
 
-			return new UnblindedSignature(new BigInteger(c), new BigInteger(s));
+			var carr = cs.ToByteArray(true, true);
+			var sarr = ss.ToByteArray(true, true);
+
+			carr = new byte[32 - carr.Length].Concat(carr).ToArray();
+			sarr = new byte[32 - sarr.Length].Concat(sarr).ToArray();
+			var signatureBytes = carr.Concat(sarr).ToArray();
+			var signature = ByteHelpers.ToHex(signatureBytes);
+
+			var sig = UnblindedSignature.Parse(signature);
+			return sig;
 		}
 
 		/// <inheritdoc />
 		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
 		{
-			UnblindedSignature signature = (UnblindedSignature)value;
+			var signature = (UnblindedSignature)value;
+			var c = new BigInteger(signature.C.ToBytes(), isUnsigned: true, isBigEndian: true);
```

Comment:
ok I am surprised it is big endian here.

Reply:
I was surprised too because you told me that sec256k1 assumes little endian but I run this and the previous version in parallel for 1 million signatures and it works as the old version.

Reply:
I checked the code and indeed, the Scalar is big endian. This will not change, as this is how secp256k1 is made.